website/redirects.js

```diff
@@ -5212,4 +5172,468 @@ module.exports = [
+    destination: "/consul/commands/v1.18.x/snapshot",
+    permanent: true,
+  },
+  {
+    source:
+      "/consul/commands/:version(v1\\.(?:19|20)\\.x)/resource",
+    destination: "/consul/commands/:version/",
+    permanent: true,
+  },
+  {
+    source: "/consul/connect/config-entries/service-intentions",
+    destination: "/consul/docs/reference/config-entry/service-intentions",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/acl/:slug*",
+    destination: "/consul/docs/secure/acl/:slug*",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/agent/checks",
+    destination: "/consul/docs/reference/service/health-check",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/agent/config-entries/:slug*",
+    destination: "/consul/docs/reference/config-entry/:slug*",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/agent/encryption",
+    destination: "/consul/docs/fundamentals/agent",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/agent/options",
+    destination: "/consul/docs/fundamentals/agent",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/api-gateway/tech-specs",
+    destination: "/consul/docs/north-south/api-gateway/k8s/tech-specs",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/api-gateway/usage",
+    destination: "/consul/docs/consul/docs/north-south/api-gateway",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/api-gateway/usage",
+    destination: "/consul/docs/consul/docs/north-south/api-gateway",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/commands/:slug*",
+    destination: "/consul/commands/:slug*",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/concepts/anti-entropy",
+    destination: "/consul/docs/concepts/consistency",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/concepts/security",
+    destination: "/consul/docs/architecture/security",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/config-entries/partition-exports",
+    destination: "/consul/docs/reference/config-entry/exported-services",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/configuration",
+    destination: "/consul/docs/fundamentals/config-entry",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/gateways/api-gateway/configuration/:slug*",
+    destination: "/consul/docs/north-south/api-gateway",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/gateways/ingress-gateway/:slug*",
+    destination: "/consul/docs/north-south/ingress-gateway",
+    permanent: true,
+  },
+  {
+    source: "/docs/k8s/crds",
+    destination: "/consul/docs/fundamentals/config-entry",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/:version(v1\\.(?:18|19|20)\\.x)/agent/telemetry",
+    destination: "/consul/docs/:version/agent/monitor/telemetry",
+    permanent: true,
+  },
+  {
+    source: "/docs/connect/intentions",
```

Comment:
Suggesting to prefix the source with `/consul`, as otherwise this redirect is invalid.

```suggestion
    source: "/consul/docs/connect/intentions",
```